fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3306,6 +3290,54 @@ class DDTeamCollectionImpl {
 
 }; // class DDTeamCollectionImpl
 
+void DDTeamCollection::updateTeamPivotValues() {
```

Comment:
It might help if we move `teamAvailableSpace` and `teamAverageCPU_index` into two separate functions, and calling them from `updateTeamPivotValues`.